environments/staging/public.yml

```diff
@@ -1,6 +1,8 @@
 DATADOG_ENABLED: True
 monitor_celery_heartbeat: False
 
+ansible_version: 4.2.0
```

Comment:
I don't like the idea of this being a per-env setting, since it's already managed by `setup.py` and `requirements.txt`. Is this needed for anything other than the newly added error message?